src/matter/cluster/BasicCluster.ts

```diff
@@ -16,9 +21,10 @@ export const BasicCluster = Cluster(
     0x28,
     "Basic",
     {
-        vendorName: Attribute(1, StringT),
+        dataModelRevision: Attribute(0, UnsignedIntT),
+        vendorName: Attribute(1, StringT), /* length: 32 */
```

Comment:
I think this is maxLength and not length.

Matter specs uses "length" for both, but in some case, it is the max length, in other it is the mandatory length for the string.
It will be important to know the different when this will be used for data verification during parsing / unparsing.

Reply:
I still left the PR as "draft" becauser I also wanted to check against the PDF specs :-)

Reply:
you were right ... I started to sync with specs ... some smaller diffs